database/migrations/2023_10_28_162444_create_attributes_table.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('attributes', function (Blueprint $table) {
+            $table->char('id', 36)->collation('ascii_bin')->primary();
+            $table->char('facility_id', 36)->collation('ascii_bin')->nullable();
+            $table->char('table');
+            $table->char('model');
+            $table->char('name');
+            $table->char('api_name');
+            $table->char('type');
+            $table->char('dictionary_id', 36)->collation('ascii_bin')->nullable();
+            $table->integer('default_order');
+            $table->boolean('is_multi_value')->nullable();
+            $table->char('requirement_level');
+
+
+
+            $table->timestamps();
+
+            $table->foreign('facility_id')
+                ->references('id')
+                ->on('facilities')
+                // Null has special meaning in this table ('any facility'), maybe it should restrict or cascade
+                //(otherwise facility-specific attributes become available to everyone).
```

Comment:
to jest dobra uwaga w sumie, w obu tych miejscach ma więcej sensu restrict. Możesz zmienić i napisać komentarze


kolumna typu słownik po usunięciu słownika tez nie jest szczególnie pełnosprawna

facility_id:  set null would mean, that facility-specific attributes become available to everyone

dictionary_id: set null would mean, that "dict" attribute loses connection to dictionary

czy jakkolwiek się to pisze po angielsku :D